x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/parser/ExpressionBuilder.java

```diff
@@ -117,7 +117,7 @@ public Literal visitIntegerValue(EsqlBaseParser.IntegerValueContext ctx) {
 
         try {
             number = StringUtils.parseIntegral(text);
-        } catch (QlClientIllegalArgumentException siae) {
```

Comment:
super-duper nit: variable name of the exception is slightly less clear now (since it referred to a type); maybe let's just replace it by `e`.

Reply:
Ah, yes, I'll update this with a next PR.